src/NovelRT/ResourceManagement/Desktop/DesktopResourceLoader.cpp

```diff
@@ -205,13 +205,14 @@ namespace NovelRT::ResourceManagement::Desktop
             returnImage.emplace_back(rawImage[i]);
         }
 
+        delete[] rawImage;
```

Comment:
Was this a leak that somehow slipped through?

Reply:
It was a possible leak since we were throwing before deletion. If someone try-catched, it'd leak.